scripts-2.0/helpers/Gravatar.js

```diff
@@ -0,0 +1,18 @@
+import md5 from 'crypto-js/md5';
+
+export default class Gravatar {
+    constructor() {
+    }
+
+    getGravatarUrlForEmail = (email, s = 120) => {
+        let url = 'https://www.gravatar.com/avatar/';
```

Comment:
to do configa może wywalmy?

Reply:
done, poszło do stałej jak ustalone na HS